### ISSUE TITLE
remove g_disabledVoteCalls

### ISSUE THREAD

**Issue by ghost** [Unvanquished/Unvanquished#1985]
This cvar is useless as votes all have a configurable ratio, and having those ratios at 0 or less will effectively disable the vote.
To have 2 ways to disable votes is imo just a good way to make things more confuse.

Also, this cvar receives words, which the user have to guess and there is no validation of those AFAICT, which makes it even more confusing.

**Comment by slipher** [comment]
Some percentage cvars are used for multiple vote types. Maybe you want to allow `kickbots` but not `kick` for example.

**Comment by bmorel** [comment]
Good point. Maybe it would be better to have individual cvars thought, instead of having  2 cvars affecting things, one of them being a list of words the admin have to guess (there is no exhaustive list of what is allowed, and no feedback that a word in there does not exists, as with the others "parse list" like cvars).

**Comment by slipher** [comment]
Now we have "custom votes" so there definitely won't be a cvar for each one. Of course the user could disable a custom vote by deleting it from `admin.dat` but that would be cumbersome; it is nice to have a quick way with a cvar. I think we will keep `g_disabledVoteCalls`.
